spanner/read.go

```diff
@@ -121,6 +130,27 @@ type RowIterator struct {
 	sawStats         bool
 }
 
+// RowsReturned returns the number of rows returned by the query. If the query
+// was a DML statement, the number of rows affected is returned. If the query
+// was a PDML statement, the number of rows affected is a lower bound.
```

Comment:
```suggestion
// RowsReturned returns, if the query was executed with query stats enabled,
// the number of rows returned by the query. 
//
// If the query was a DML statement, the number of rows affected is returned. 
// If the query was a PDML statement, the number of rows affected is a lower bound.
// If the query was executed without query stats enabled, or if it is otherwise
// impossible to determine the number of rows in the resultset, -1 is returned.
```